tests/unit/tests_sonata.cpp

```diff
@@ -278,3 +278,78 @@ BOOST_AUTO_TEST_CASE( genericMVDFile )
     BOOST_CHECK_EQUAL(neurons_pos[0][1], static_cast<double>(1986.506637));
     BOOST_CHECK_EQUAL(neurons_pos[0][2], static_cast<double>(10.788424));
 }
+
+
+BOOST_AUTO_TEST_CASE( newSonataTestEmodel )
+{
+    SonataFile file(SONATA_FILENAME_NEW_FORMAT);
+
+    std::vector<std::string> emodels;
+
+    emodels = file.getEmodels();
```

Comment:
`std::vector<std::string> emodels{file.getEmodels()};` ?
A lot in this file could be changed like that

Reply:
I'm gonna be the black sheep here in the end... :p I don't know if under some circumstances the copy elision won't happen with this syntax. My suggestion is the short and standard
auto emodels = getEmodels()